docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -48,52 +41,64 @@ The component can be uncontrolled or controlled
+The component is available in four variants:
 
-- The `MobileNextDatePicker` component works best for touch devices and small screens.
-- The `DesktopNextDatePicker` component works best for mouse devices and large screens.
+- The `DesktopNextDatePicker` component which works best for mouse devices and large screens.
+  It renders the views inside a popover and allows editing values directly inside the field.
 
-By default, the `NextDatePicker` component renders the desktop version if the media query [`@media (pointer: fine)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) matches.
-This can be customized with the `desktopModeMediaQuery` prop.
+- The `MobileNextDatePicker` component which works best for touch devices and small screens.
+  It renders the view inside a modal and does not allow editing values directly inside the field.
 
-There are certain caveats when testing pickers, please refer to [this section](/x/react-date-pickers/getting-started/#testing-caveats) for more information.
+- The `NextDatePicker` component which renders `DesktopNextDatePicker` or `MobileNextDatePicker` depending on the device it runs on.
 
-{{"demo": "ResponsiveDatePickers.js"}}
+- The `StaticDatePicker` component which renders without the popover/modal and field.
 
-## Static mode
+{{"demo": "ResponsiveDatePickers.js"}}
 
-It is also possible to render the Date Picker without the modal/popover and text field.
-This can be helpful when building custom popover/modal containers.
+By default, the `NextDatePicker` component renders the desktop version if the media query [`@media (pointer: fine)`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer) matches.
+This can be customized with the `desktopModeMediaQuery` prop.
 
-{{"demo": "StaticDatePickerDemo.js", "bg": true}}
+:::warning
+Responsive components can suffer some inconsistencies between testing environments if media query is not supported.
+Please refer to [this section](/x/react-date-pickers/getting-started/#testing-caveats) for solutions.
+:::
 
 ## Form props
 
-The Date Picker component can be disabled or read-only.
+The component can be disabled or read-only.
 
 {{"demo": "FormPropsDatePickers.js"}}
 
 ## Views
 
-The component can contain three views: `day`, `month`, and `year`.
-By default, only the `day` and `year` views are enabled.
+The component supports three views: `day`, `month`, and `year`.
 
-You can customize the enabled views using the `views` prop.
-Views will appear in the order they're included in the `views` array.
+By default, the `day` and `year` views are enabled.
+Use the `views` prop to change this behavior:
 
 {{"demo": "DatePickerViews.js"}}
 
-## Landscape orientation
+By default, the component renders the `day` view on mount.
+Use the `openTo` prop to change this behavior:
 
-By default, the Date Picker automatically sets the orientation based on the `window.orientation` value.
+{{"demo": "DatePickerOpenTo.js"}}
+
+:::info
```

Comment:
This callout would probably make more sense as `success` as [per Sam's recommendation](https://www.notion.so/mui-org/Callouts-and-their-usage-in-the-docs-7e709a02c72142cd8b1a0829861435c5).

Reply:
I'll apply it to the other pages
I wanted a first opinion on it, but it's just a few pages, if somebody wants a change I'll apply it on all pages :+1: